scripts/model/react-client-scr-answers.json

```diff
@@ -17,7 +14,7 @@
     "entity": {
       "name": "scr$Car"
     },
-    "managementComponentName": "CarManagement"
+    "managementComponentName": "Car Management Cards"
```

Comment:
It ends up as a class name so I think we need to make the generator to remove spaces. This way we will end up with  both a proper human-readable name in i18n file and a proper class name.

```
export class Car Management Cards extends React.Component<Props>
```

Reply:
This change removed from this branch. Will be implemented in sepatrate issue.